apps/app/src/test/scala/org/lfdecentralizedtrust/splice/integration/tests/TimeBasedTestNetPreviewIntegrationTest.scala

```diff
@@ -36,9 +36,14 @@ class TimeBasedTestNetPreviewIntegrationTest
         }
       },
     )(
-      "Wait for SV rewards to be collected",
+      "Wait for SV rewards for rounds 1&2 to be collected",
       _ => {
         sv1WalletClient.balance().unlockedQty should be > BigDecimal(0)
+        (1 to 2).foreach { _ =>
+          eventually() {
+            ensureNoSvRewardCouponExistsForRound(_, sv1WalletClient)
+          }
+        }
```

Comment:
```suggestion
        forAll(Seq(1,2)) { round =>
            ensureNoSvRewardCouponExistsForRound(round, sv1WalletClient)
        }
```

forAll usually gives better errors when it fails. And the eventually seems redundant given that you are already in an actAndCheck